config/config/sc-config.yaml

```
@@ -175,31 +215,74 @@ elasticsearch:
       limits:
         memory: 1024Mi
         cpu: 1
-  storageClass: local-storage
-  retention:
-    kubeAuditSize: 50
-    kubeAuditAge: 30
-    kubernetesSize: 50
-    kubernetesAge: 50
-    otherSize: 10
-    otherAge: 30
+    affinity:
+      podAntiAffinity:
+        preferredDuringSchedulingIgnoredDuringExecution:
+        - weight: 1
+          podAffinityTerm:
+            topologyKey: "kubernetes.io/hostname"
+            labelSelector:
+              matchLabels:
+                role: client
+    tolerations: []
+    nodeSelector: {}
+  # Config for https://www.elastic.co/guide/en/elasticsearch/client/curator/5.8/about.html
+  curator:
+    enabled: true
+    resources:
+      limits:
+        cpu: 100m
+        memory: 128Mi
+      requests:
+        cpu: 10m
+        memory: 32Mi
+    affinity: {}
+    nodeSelector: {}
+    tolerations: []
+    retention:
+      kubeAuditSize: 50
+      kubeAuditAge: 30
+      kubernetesSize: 50
+      kubernetesAge: 50
+      otherSize: 10
+      otherAge: 30
+      # (Optional) retention for indices matched by 'postgresql-*'
+      # postgresql: false
+      # postgresqlSize: 30
+      # postgresqlAge: 30
+  # Index state management
+  ism:
+    # Overwrite ism policies
+    updatePolicies: true
```

Could we rename this to `overwritePolicies` to match `overwriteTemplates` below?
```suggestion
    overwritePolicies: true
```

---

Yes we can